MAINTAINERS.yml

```diff
@@ -217,6 +217,7 @@ Bluetooth Mesh:
 Build system:
     status: maintained
     maintainers:
+        - jeremybettis
```

Comment:
is this intentional? should be collaborator?

Reply:
My mistake.  Fixed in the latest PR.